### PR TITLE
Bug 1117972 - Show search notice again

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c6a881b273c8)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=94a08fd70ad5)

File: apps/search/js/providers/suggestions.js [hunk 1]
```diff
@@ -46,6 +46,10 @@
 
     remote: true,
 
+    clear: function() {
+      this.container.innerHTML = '';
+    },
```
Comment (benfrancis) [inline apps/search/js/providers/suggestions.js]: What's this doing here? Doesn't this get inherited from the provider base class?

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f72262eb8e0e)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fc433dd32ff4)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a2bb61697afd)

File: apps/search/js/providers/suggestions.js (hunk 1 above)
Comment (daleharvey) [inline apps/search/js/providers/suggestions.js]: Yeh for some reason I was getting unit test failures previous to adding it, but must have been something else, passing now without it

**Pull request closed without merging**